js/mixins/geometry.js

```diff
@@ -0,0 +1,17 @@
+/**
```

Comment:
Is it necessary to place this code in a separate file, used only once? It's basically just one line of code. We're doing the same operation inline in for example tooltip placement, click handling and series label.